mathics/builtin/arithmetic.py

```diff
@@ -1861,6 +1861,60 @@ class Factorial(PostfixOperator, _MPMathFunction):
     mpmath_name = "factorial"
 
 
+class Factorial2(PostfixOperator, _MPMathFunction):
+    """
+    <dl>
+      <dt>'Factorial2[$n$]'
+      <dt>'$n$!!'
+      <dd>computes the double factorial of $n$.
+    </dl>
+
+    >> 5!!
+     = 15
+
+    >> Factorial2[-3]
+     = -1
+
+    'Factorial2' also accept irrational numbers:
```

Comment:
Small grammar typo here and below "also accept" -> "also accepts"

Reply:
Again, thanks!